packages/client/lib/AuthorizationCodeClient.ts

```diff
@@ -113,13 +113,16 @@ export const createAuthorizationRequestUrl = async ({
   const { redirectUri, requestObjectOpts = { requestObjectMode: CreateRequestObjectMode.NONE } } = authorizationRequest;
   const client_id = clientId ?? authorizationRequest.clientId;
 
-  let { scope, authorizationDetails } = authorizationRequest;
-  const parMode = endpointMetadata?.credentialIssuerMetadata?.require_pushed_authorization_requests
+  // Authorization server metadata takes precedence
+  const authorizationMetadata = endpointMetadata.authorizationServerMetadata ?? endpointMetadata.credentialIssuerMetadata
+
+  let { authorizationDetails } = authorizationRequest;
```

Comment:
Is there a specific reason why you removed `scope` here? You are now using `authorizationRequest.scope` a few times below. Nothing big, just easier to read

Reply:
It was complaining about let vs const. So chose this over 2 destructurings or an eslint ignore. Happy to change it though

Reply:
change to const would have fixed it right :P

Reply:
No because authorizationDetails is reassigned.

So then it should have been

```
const { scope } = authorizationRequest
let { authorizationDetails } = authorizationRequest
```